shared/models/git_repo.rb

```diff
@@ -164,8 +164,8 @@ def git_and_remote_branches_each(&each_block)
         logger.debug("iterating through all remote branches of #{self.git_config.git_url}")
         branch_count = 0
         self.git.branches.remote.each do |branch|
-          each_block.call(self.git, branch)
-          branch_count = branch_count + 1
+          yield(self.git, branch)
```

Comment:
I don't agree with rubocop on this one for `each_block.call` to `yield`

Reply:
Okay, I can change the `.rubocop.yml` file 👍

Reply:
Awesome, thank you!